exampleSite/content/about-split/index.md

```diff
@@ -1,16 +1,15 @@
 +++
 title = "About Syna"
-lastmod = "2017-10-05"
 date = "2017-10-05"
 subtitle = ""
 layout = "split"
 
 [sidebar]
     title = "Sidebar"
     content = """
-So much information  
-Phone numbers  
-Details  
+So much information
+Phone numbers
+Details
```

Comment:
The trailing whitespace is usually rendered as new-line in markdown.
Removing it here leads to it being on one line and breaking the sidebar.

Reply:
Fixed in 32628bbc105e8fe69438381bb8f470c43291fcbe. My editor removes trailing whitespace.